tests/tasks/test_custom_backend.py

```diff
@@ -68,3 +83,49 @@ def test_no_enqueue(self):
             "without an implementation for abstract method 'enqueue'",
         ):
             test_tasks.noop_task.using(backend="no_enqueue")
+
+
+def custom_task_func():
```

Comment:
Suggestion: reuse a task from `./tasks.py` to avoid needing to create a new one.